pkgs/development/python-modules/immutabledict/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "immutabledict";
+  version = "2.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ylnddj3x1zvak4fp69bk2c56k3zbfgrf2q5jk9kvlj61zrvhgv7";
+  };
+
+  # No tests in PyPI archive
+  doCheck = false;
+
+  meta = with lib; {
```

Comment:
Add yourself to the maintainers, if you don't mind.

Reply:
We won't accept packages without maintainers.